docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -1056,6 +1056,19 @@ When I save number of open tabs to SCENARIO variable `count`
 
 == Tips & Tricks
 
+=== Validate URL of the opened page
+
+To validate URL of the current page the following statement can be used:
+
+.Validate URL of "https://example.com/relative/url/example.html"
+[source,gherkin]
+----
+Given I am on page with URL `https://example.com/relative/url/example.html`
+Then `${current-page-url}` is equal to `https://example.com/relative/url/example.html`
+----
+
+* `${current-page-url}` is the <<Current page URL,current page URL>> dynamic variable.
```

Comment:
I believe this is already described in section of current page URL dynamic variable

Reply:
removed